api/envoy/api/v2/auth/cert.proto

```diff
@@ -413,6 +413,9 @@ message DownstreamTlsContext {
   // certificate.
   google.protobuf.BoolValue require_client_certificate = 2;
 
+  // If specified, Envoy will request a client certificate.
+  google.protobuf.BoolValue request_client_certificate = 7;
```

Comment:
Certificate is always requested if `trusted_ca` is specified, so this is a no-op.